src/components/app.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Axios from 'axios';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import SearchForm from './search-form';
+import '../styles/app.scss';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        country: '',
+      },
+    };
+    this.handleForecastSelect = this.handleForecastSelect.bind(this);
```

Comment:
Binding methods works, but you save a bit of time is you declare them with arrow syntax. `handleForecastSelect() {}` would just be `handleForecastSelect = () => {}` with no need for these binding lines.